ROADMAP.md

```diff
@@ -0,0 +1,98 @@
+Ansible Roadmap
+=============
+This document is now the location for published Ansible Core roadmaps.     
+
+The roadmap will be updated by version. Based on team and community feedback, an initial roadmap will be published for a major or minor version (2.0, 2.1).  Subminor versions will generally not have roadmaps published.
+
+This is the first time we've published this and asked for feedback in this manner.  So feedback on the roadmap and the new process is quite welcome.  We are aiming for further transparency and better inclusion of both community desires and submissions.  
+
+These are our *best guess* roadmaps based on our own experience and based on requests and feedback from the community.  There are things that may not make it on due to time constraints, lack of community maintainers, etc.  And there may be things we missed, so each roadmap is published both as an idea of what is upcoming in Ansible, and as a medium for seeking further feedback from the community. Here are the good places for you to submit feedback:
+
+  * Our google-group: ansible-devel
+  *  Ansible Fest conferences.  
+  * IRC: Our freenode channel #ansible (this one may have things lost in lots of conversation, so a caution).
+
+2.1 Roadmap, Targeted for the End of April
+==========
+## Windows, General
+* Figuring out privilege escalation (runas w/ username/password)
+* Implement kerberos encryption over http
+* pywinrm conversion to requests (Some mess here on pywinrm/requests. will need docs etc.)
+* NTLM support
+
+## Modules
+* Windows
+  * Finish cleaning up tests and support for post-beta release
+  * Strict mode cleanup (one module in core)
+  * Domain user/group management
+  * Finish win\_host and win\_rm in the domain/workgroup modules. 
+     * Close 2 existing PRs (These were deemed insufficient)
+  * Replicate python module API in PS/C# (deprecate hodgepodge of stuff from module_utils/powershell.ps1)
+* Network
+  * Cisco modules (ios, iosxr, nxos, iosxe)
+  * Arista modules (eos)
+  * Juniper modules (junos)
+  * OpenSwitch
+  * Cumulus
+  * Dell (os10) - At risk
+  * Netconf shared module
+  * Hooks for supporting Tower credentials
+* VMware (This one is a little at risk due to staffing. We're investigating some community maintainers and shifting some people at Ansible around, but it is a VERY high priority).
+  * vsphere\_guest brought to parity with other vmware modules (vs Viasat and 'whereismyjetpack' provided modules)
+  * VMware modules moved to official pyvmomi bindings
+  * VMware inventory script updates for pyvmomi, adding tagging support
+* Azure (Notes: This is on hold until microsoft releases a working code generator.  We have basic modules working against all of these resources.  Could ship it against current SDK, but may break. Or should we pin the version?)
```

Comment:
how about: 

"This is on hold until Microsoft swaps out the code generator on the Azure Python SDK, which may introduce breaking changes. We have basic modules working against all of these resources. Could ship it against current SDK, but may break. Or should we pin the version?"